src/views/Products.jsx

```diff
@@ -1,7 +1,104 @@
-import React from 'react'
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import ProductsSidebar from '../components/ProductsSidebar'
+import ProductCategories from '../mocks/en-us/product-categories.json'
+import ProductsJson from '../mocks/en-us/products.json'
+import '../styles/Products.scss'
+import ProductItem from '../components/FeaturedProducts/ProductItem'
+import ProductsPagination from '../components/ProductsPagination'
+import Spinner from '../components/Spinner'
+
+export default function Products ({ onHandleLinkPage }) {
+  const [isLoading, setIsLoading] = useState(true)
+  const [filters, setFilters] = useState([])
+  const [filteredProducts, setFilteredProducts] = useState(ProductsJson.results)
+
+  const isInFilters = (value) => {
+    return filters.indexOf(value) !== -1
+  }
```

Comment:
this function is being assigned a new value with every rerender of the `Products` component, the hook `useCallback` can help yo memoize the function value and assigned the new value only when it is necessary